### PR TITLE
[ONNX] Update ONNX landing page since 1.3

### PR DESCRIPTION
**Pull request by BowenBao**
* New ops supported for exporting.
* Updates on support for tensor indexing and dynamic list of tensors.
* @lara-hdr, @spandantiwari Should we also include updates on torchvision support in this page?

cc @houseroad, @neginraoof Please review if I have missed anything.

### CODE REVIEW

File: docs/source/onnx.rst [hunk 1]
```diff
@@ -705,6 +654,66 @@ Q: Does ONNX support implicit scalar datatype casting?
...
+    out_ort = sess.run(None, {
+        sess.get_inputs()[0].name: torch.zeros(3, 4).numpy(),
+        sess.get_inputs()[1].name: new_data.numpy(),
+    })
+
+    assert torch.all(torch.eq(out, torch.tensor(out_ort)))
+
+Q: Is tensor list exportable to ONNX?
+
+  Yes, this is supported now for ONNX opset version >= 11. E.g.: ::
```
Comment (houseroad) [inline docs/source/onnx.rst]: Do you want to mention how we express tensor list in ONNX?

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit 8974e014:


**None of the build failures appear to be your fault**.

* **1/1** [broken upstream](https://dr.pytorch.org/master-timeline.html?count=1&sha1=b565d9b35696259f9ab7d5dfd3814c82d40aaa84&should_suppress_scheduled_builds=true&should_suppress_fully_successful_columns=true&max_columns_suppress_successful=35) at merge base b565d9b3 since Jan 29
  <details>
  <summary>
  Please rebase on the <code>viable/strict</code> branch (expand for instructions)
  </summary>
  
  If your commit is newer than `viable/strict`, you can try basing on an older, stable commit:
  ```
  git fetch origin viable/strict
  git rebase --onto viable/strict $(git merge-base origin/master HEAD)
  ```
  If your commit is older than `viable/strict`:
  ```
  git fetch origin viable/strict
  git rebase viable/strict
  ```
  Check out the [recency history](https://dr.pytorch.org/master-viable-commits.html) of this "viable master" tracking branch.
  
  </details>
  


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=8974e0149e5b180e86833add0c758cb889b44a15).

### :construction: 1 upstream failure recognized by patterns:
These builds matched patterns, but were probably **caused by upstream breakages**:
* [pytorch_linux_backward_compatibility_check_test](https://dr.pytorch.org/build-details.html?build_id=92966619) from Jan 29



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

**Review by lara-hdr — APPROVED** [review]
LGTM. Thanks!

**Review by houseroad — APPROVED** [review]
LGTM, thanks!

**Pull request closed without merging**